### DISCUSSION TITLE
client building fails

### DISCUSSION THREAD

**Discussion by ccfriend** [alvr-org/ALVR#973] [General]
when build client, I encounter this error, I am building on Windows11 and PowerShell. And now I have no idea why this happened.  

= note: clang: error: unable to execute command: program not executable
          clang: error: linker command failed with exit code 1 (use -v to see invocation)
          C:\work\ALVR>"python" "C:\work\ALVR\alvr\client\android\build\linker-wrapper\linker-wrapper.py" @C:\Users\cheng\AppData\Local\Temp\rustcZU8fkC\linker-arguments
          'C:\Users\cheng\AppData\Local\Android\Sdk\ndk\22.0.7026061\toolchains\llvm\prebuilt\windows-x86_64\bin\aarch64-linux-android24-clang.cmd' -Wl,-soname,libalvr_client.so '@C:\Users\cheng\AppData\Local\Temp\rustcZU8fkC\linker-arguments'

**Comment by thisleo** [comment]
you shuold install rust, download from [here](https://www.rust-lang.org/learn/get-started), then run rustup-init.exe
